android/app/src/main/java/im/status/ethereum/MainApplication.java

```diff
@@ -32,7 +32,7 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
-import java.util.function.Function;
+import im.status.ethereum.function.Function;
```

Comment:
I don't where this is

Reply:
this lives in 
`compile 'status-im:function:0.0.1' `
ant this is maven project which literally contains only
```java
package im.status.ethereum.function;

public interface Function<T, R> {
    R apply(T var1);
}
```
i would prefer duplicate this piece of code in all project, but gradle will not build it...